components/sections/carousel/carousel.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import Marquee from 'react-fast-marquee';
+import { Image } from 'components/shared';
+
+const Carousel = ({ data }) => {
+  const { logos } = data;
+
```

Comment:
Maybe worth to also have a sanity check to prevent empty `logos`


```suggestion
      if (!logos.length) return null;

```